src/extension/css/src/main/java/org/geoserver/community/css/web/CssHandler.java

```diff
@@ -5,6 +5,7 @@
  */
```

Comment:
Header copyrigth was not updated. Kevin shared a nice script to do that on all files in a commit, in one shot, please do so :)